packages/lesswrong/lib/collections/clientIds/collection.ts

```diff
@@ -20,7 +20,7 @@ addUniversalFields({
   createdAtOptions: {canRead: ['admins']},
 });
 
-ensureIndex(ClientIds, {clientId: 1});
+ensureIndex(ClientIds, {clientId: 1}, {unique: true, concurrently: true, name: "idx_ClientIds_clientId_unique"});
```

Comment:
The reason for the explicit name is that the generated name doesn't include whether the index is unique or not, so it says the index already exists. Ideally we would change `CreateIndexQuery` so it does include this but making this backwards compatible (i.e. not duplicating all existing unique indexes) would be a pain so I haven't done it for this PR